src/utils/getTextTitleAndContent.js

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios'
+const apiUrl = 'http://localhost:3000/texts'
+
+catch(error){
+    console.error('There was an error.', error);
+    return null;
+}
+}
+
```

Comment:
there seems to be a missing function here. Where is the getTextTitleAndContent function?